src/channeldef.ts

```diff
@@ -61,7 +64,9 @@ export type ChannelDefWithCondition<F extends FieldDef<any>, V extends Value> =
 /**
  * @minProperties 1
  */
-export type ValueDefWithCondition<F extends FieldDef<any>, V extends Value = Value> = Partial<ValueDef<V>> & {
+export type ValueDefWithCondition<F extends FieldDef<any>, V extends ValueOrGradient = ValueOrGradient> = Partial<
```

Comment:
I'm wondering if the default should be `= ValueOrGradient` or ` = Value`?  I'm a bit worried that this might lead to mistakes when we add more channels. That said, if we have to override the default in too many internal codes, I'm fine with this as the default.

Reply:
Done